packages/extension/src/background/index.ts

```diff
@@ -118,7 +121,19 @@ browser.browserAction.onClicked.addListener(() => {
   browser.tabs.create({ url, active: true });
 });
 
-browser.runtime.onInstalled.addListener(async () => {
+browser.runtime.onInstalled.addListener(async (details) => {
+  const data = getCompanionAlerts();
```

Comment:
Should we generalize it perhaps? Not just for companion alerts but every alert we possibly need and then expose it in a context. We can do it later as well but just change the name of the storage key.
cc @rebelchris

Reply:
Very good point, if we can generalize this to be non companion related we and in it's own scope we are there :)